src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operationType = parts[0].trim();
+                int amount = Integer.parseInt(parts[1].trim());
+
+                if (operationType.equals("supply")) {
+                    totalSupply += amount;
+                } else if (operationType.equals("buy")) {
+                    totalBuy += amount;
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Consider logging the exception or providing a user-friendly message instead of just printing the stack trace. This will make it easier to understand what went wrong if an error occurs.